### ISSUE TITLE
Unpitched Percussion Wishlist

### ISSUE THREAD

**Issue by brandonbrookspercussion** [musescore/MuseScore#16707]
### Issue type

Muse Sounds bug

### Bug description

The MuseSound temple blocks sounds only have 2 “pitches”. Temple blocks have 5. 

### Steps to reproduce

No steps needed. Incorrect instrument range. 

### MuseScore Version

MuseScore 4.0.1

### Regression

No.

### Operating system

Monterey 12.6.3

**Comment by brandonbrookspercussion** [comment]
Also it sounds like the temple block sound and the wood block sound may be reversed.

**Comment by PalgamingHD** [comment]
Not just a muse sounds issue but also a notation issue with musescore itself*

**Comment by brandonbrookspercussion** [comment]
> Not just a muse sounds issue but also a notation issue with musescore itself*

Correct. I should have mentioned that. 🤦‍♂️

Typically they are written in a regular staff on the lines or spaces (starting with D below the staff or ending on G above the staff). 

Percussion notation is very fluid from composer to composer. Lol.

**Comment by PalgamingHD** [comment]
For sure, I like writing them on each line. I feel like there should be a quick way to assign stuff like this in any given space or line in an "aux percussion" part to different sounds instead of using a look up table for midi percussion.

**Comment by itaranto** [comment]
I think this is not a sound issue, it happens with any sound font (I'm not using Muse Sound).

For example, Wood Blocks which have only 2 "parts", have the same pitch.

This breaks a lot of scores I made with MS3, this *is* a regression IMHO and it should have higher priority.

**Comment by shoogle** [comment]
Can we clarify, are there really three separate issues here?

1. Temple blocks should have a 5 line staff (bug).
2. Add sound for Temple Blocks in Muse Sounds (feature request).
3. Wood block uses the same sound for high and low note in MS Basic (bug).

**Comment by itaranto** [comment]
> Can we clarify, are there really three separate issues here?
> 
>     1. Temple blocks should have a 5 line staff (bug).
> 
>     2. Add sound for Temple Blocks in Muse Sounds (feature request).
> 
>     3. Wood block uses the same sound for high and low note in MS Basic (bug).

Do you want me to open a separate issue for the Wood Block (and similar) pitch issues?

**Comment by shoogle** [comment]
@itaranto, yes please! The issue can include any unpitched (according to MuseScore) percussion instruments that are using the wrong sound with MS Basic.

**Comment by brandonbrookspercussion** [comment]
I agree that it would be great to assign any sound to what ever line. If you have a multi-percussion set up, it could include literally anything. That's different on every piece of music. It could be notated just fine but assigning the appropriate sounds won't currently work. You would have to have multiple staves to notate something that could simply be placed in one stave.

**Comment by shoogle** [comment]
@brandonbrookspercussion, that's already possible via the 'Edit drumset' button when in note input mode on a percussion staff.

![image](https://user-images.githubusercontent.com/11011881/229310610-18671e51-e133-4781-80cc-e79609dc2908.png)

Caveats:

- The interface is horrible
- It doesn't work with Muse Sounds

But you can expect both of those things to be addressed at some point in the not too distant future.

**Comment by brandonbrookspercussion** [comment]
Thank you!

**Comment by PalgamingHD** [comment]
Maybe I could also add some other percussion related issues, the whistle is treated like a non sustaining instrument, whip does not have any sort of sound, and in general using general midi mapping makes it a pain to make any sort of custom sounds or any other percussion instruments. Maybe a musescore sampler is needed to input sounds and play them back as notes

**Comment by shoogle** [comment]
@brandonbrookspercussion, we're aware of VDL and we would indeed like to make it easier to use with MuseScore in the future so users don't have to look up pitch numbers and enter them manually, etc.

@PalgamingHD 

> the whistle is treated like a non sustaining instrument

I think that's a limitation of General MIDI

> using general midi mapping makes it a pain to make any sort of custom sounds or any other percussion instruments

We're aware. Our future percussion sounds may not be based on MIDI, though MIDI will remain as an option.

> whip does not have any sort of sound

Probably worth raising a separate issue about that. We can assign any random sound that is remotely similar to a whip until we have something better.

**Comment by brandonbrookspercussion** [comment]
❤️
